packages/ipfs-http-server/src/api/resources/block.js

```diff
@@ -69,7 +69,7 @@ exports.get = {
       throw Boom.notFound('Block was unwanted before it could be remotely retrieved')
     }
 
-    return h.response(block.data).header('X-Stream-Output', '1')
+    return h.response(Buffer.from(block.data.buffer)).header('X-Stream-Output', '1')
```

Comment:
This is a bit strange - Hapi is node-only so you can sort of see why it'd only accept a bonafide `Buffer` object but you'd think it'd be able to handle plain Uint8Arrays.

At the very least here we should use the three-arg version to make sure we're doing a no-copy operation:

```suggestion
    return h.response(Buffer.from(block.data.buffer, block.data.byteOffset, block.data.byteLength)).header('X-Stream-Output', '1')
```